vowpalwabbit/cbify.cc

```diff
@@ -173,7 +394,7 @@ void predict_or_learn(cbify& data, single_learner& base, example& ec)
     base.learn(ec);
 
   data.a_s.clear();
-  data.a_s = ec.pred.a_s;
+  data.a_s = ec.pred.a_s;  // TODO: the above line needs to be moved to after this line!
```

Comment:
Can you address this TODO before merging?

Reply:
Done.